### PR TITLE
#4882 [Manager] Event log relative minor bugs

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4882)
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/EventLog/EventLogList.js [hunk 1]
```diff
@@ -33,7 +33,11 @@ const MemorizeList = React.memo(
       rowRenderer={EventLogRow}
     />
   ),
-  (prevProps, nextProps) => isEqual(prevProps.data, nextProps.data),
+  (prevProps, nextProps) =>
+    isEqual(prevProps.data, nextProps.data) &&
+    // Event if data is not changed
```
Comment (konekoya) [inline ohara-manager/client/src/components/EventLog/EventLogList.js:38]: Even :)

Comment (konekoya) [reply ohara-manager/client/src/components/EventLog/EventLogList.js:38]: Good catch by the way.

File: ohara-manager/client/src/store/epics/eventLog/fetchEventLogsEpic.js [hunk 2]
```diff
@@ -14,10 +14,16 @@
  * limitations under the License.
  */
 
-import { get } from 'lodash';
+import * as _ from 'lodash';
```
Comment (konekoya) [inline ohara-manager/client/src/store/epics/eventLog/fetchEventLogsEpic.js:17]: `import _ from 'lodash';` will do

File: ohara-manager/client/src/store/reducers/entities/eventLogs.js [hunk 3]
```diff
@@ -44,6 +44,12 @@ const initialState = {
 
 export default function reducer(state = initialState, action) {
   switch (action.type) {
+    case actions.fetchEventLogs.TRIGGER:
+      return {
+        ...state,
+        data: [],
+        isFetching: true,
```
Comment (eechih) [inline ohara-manager/client/src/store/reducers/entities/eventLogs.js:51]: redux state 有區分 entities 及 ui，只有 data 屬於 entities，其餘應該搬到 ui。

**Pull request closed without merging**